internal/gcsx/random_reader.go

```diff
@@ -350,5 +370,25 @@ func (rr *randomReader) startRead(
 	rr.start = start
 	rr.limit = end
 
+	captureMetrics(ctx, rr.seeks)
```

Comment:
I think we can also record the size (end - start) of the read? Or is that been done elsewhere?

Reply:
We have gcsfuse_read_bytes metrics which captures the read bytes. Not based on data fetched from gcs but data read by user.  do you want to capture the amount of data fetched from gcs?

Reply:
right, that's the intention.